lib/hooks/useBrowseResultFocus/useBrowseResultFocus.js

```diff
@@ -0,0 +1,24 @@
+import {
+  useEffect,
+  useRef,
+} from 'react';
+
+const useBrowseResultFocus = isLoading => {
+  const isPaginationClicked = useRef(false);
+  const resultsContainerRef = useRef(null);
+
+  useEffect(() => {
```

Comment:
Inside MCLRenderer there already is logic to handle focus - probably there's some mistake in indexes

Reply:
`Browse` search can return 95 results for the first page and 100 for the second one - this behavior can't be handled in MCLRenderer. Actually, the `next` behavior can be fixed in `MCLRenderer`, but the `prev` one can't.

Reply:
In this case can we pass length of results instead of constant in BrowseRoute?

Reply:
```
const remainder = rowIndex % pageAmount;

const rowIndexForPrevButton = remainder === 0
  ? rowIndex - (pageAmount * 2)
  : rowIndex - (pageAmount + remainder);
```
                          
Let's calculate the focused index for the case when the response comprises 100 results and `prev` field. The `prev` field means that there is some number of results on the previous page. The `rowIndex` will be equal to the number of results (100). After clicking the Previous button the focused row will be:

```
 const remainder = rowIndex % pageAmount = 100 % 100 = 0;
 const rowIndexForPrevButton = rowIndex - (pageAmount * 2) = 100 - (100 * 2) = -100
 ```